webpack.config.cjs

```diff
@@ -27,8 +27,9 @@ module.exports = {
                      // @author       double-beep
                      // @contributor  Xnero
                      // @description  Watch, blacklist and see domain stats directly from the FIRE popup!
-                     // @match        https://chat.stackexchange.com/rooms/11540/charcoal-hq
-                     // @match        https://chat.stackexchange.com/transcript/11540*
+                     // @match        *://chat.stackexchange.com/rooms/11540/charcoal-hq?*
```

Comment:
```suggestion
                     // @match        *://chat.stackexchange.com/rooms/11540/charcoal-hq*
```

Didn't see this earlier, why is `?` needed? It prevents the userscript from running in `https://chat.stackexchange.com/rooms/11540/charcoal-hq`.